go/parquet/pqarrow/schema.go

```diff
@@ -388,7 +388,7 @@ func fieldToNode(name string, field arrow.Field, props *parquet.WriterProperties
 		}
 		return schema.MapOf(field.Name, keyNode, valueNode, repFromNullable(field.Nullable), -1)
 	default:
-		return nil, xerrors.New("not implemented yet")
+		return nil, xerrors.Errorf("support for %v not implemented yet", field.Type.ID())
```

Comment:
probably makes more sense to just do `field.Type` instead of `field.Type.ID()` which would be more generic (it would give "MAP" instead of "Map<string, int>" etc....

Reply:
The `switch` that this is the default for is on `field.Type.ID()`, which is why I thought it makes more sense to use `field.Type.ID()` in the error as well. WDYT?

Reply:
okay, that makes sense. I tend to like being more specific but I agree that since that's what we're switching on, it's good to be clear rather than expecting a consumer to understand the difference.